torch/testing/_internal/common_utils.py

```diff
@@ -1405,6 +1405,24 @@ def printErrors(self) -> None:
 IS_ARM64 = platform.machine() in ('arm64', 'aarch64')
 IS_S390X = platform.machine() == "s390x"
 
+def get_nvidia_gpu_model() -> str:
+    """
+    Retrieves the model of the NVIDIA GPU being used.
+    Will return the name of the first GPU listed.
+    Returns:
+        str: The model of the NVIDIA GPU or empty str if not found.
+    """
+    try:
+        model = subprocess.check_output(
```

Comment:
You should just be able to use the existing information that doesn't depend on subprocess. See the blackwell tests in test_max_autotune.py.